README.md

```diff
@@ -1,33 +1,51 @@
 # engineering-education
-A community-generated pool of educational content which is useful for engineers of every level. Published content can be found at [https://www.section.io/engineering-education/](https://www.section.io/engineering-education/).
+The Engineering Education Program at Section was created as a medium to provide students a friendly and inviting place to contribute to real-world projects to help students better build up their professional portfolios and profiles (upon entering the DEV career space). 
```

Comment:
I think we should highlight the notion of providing a friendly and inviting place to connect with like-minded peers who are preparing for careers in computer science-related fields.

In terms of 'contributions to real-world projects', is this referring to app enhancement contributions? Seems a bit overstated, at least for the current state of the program.